src/app/components/ActionCards/ActionCards.tsx

```diff
@@ -8,6 +8,8 @@ import {
   Box,
 } from '@mui/material';
 import { useProducts } from '../../../../lib/services/service';
+import { Product } from '@/app/models/product';
```

Comment:
jeśli gdzieś robisz takie ścieżki - to utrzymałbym konwencje i zmienił to wszędze z relatywnych :)